lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
```

Comment:
Derp, by screenshots I thought you meant screenshots of the highlighted recommendation, for verifying this patch works across platforms. Sorry I wasn't clear about that. Don't worry about screenshotting code, just go ahead and push your changes.